pyproject.toml

```
@@ -84,6 +84,8 @@ aiofiles = "^24.1.0"
 # Detected incompatible Protobuf Gencode/Runtime versions when loading authzed/api/v1/core.proto: gencode 5.28.2 runtime 5.27.3.
 protobuf = "^5.29.3"
 poetry = "^2.1.1"
+python-box = ">=7.0.1"
```

```suggestion
python-box = "^7.0.1"
```
I try to use the `^` and `~` for specifiers where possible, `^` means `>=7.0.1,<8`.
this will break if kr8s ever updates to 8 but since we use it in our code, that's a good signal to double check that things still work and manually bump

---

**done**